numpy/core/src/umath/simd.inc.src

```diff
@@ -55,6 +55,40 @@ abs_ptrdiff(char *a, char *b)
     return (a > b) ? (a - b) : (b - a);
 }
 
+/*
+ * nomemoverlap - Do two strided arrays overlap?
+ * Need to accomodate negative strides.
+ */
+static NPY_INLINE npy_bool
+nomemoverlap(char *ip,
+             npy_intp ip_size,
+             char *op,
+             npy_intp op_size)
+{
+    char *ip_start, *ip_end, *op_start, *op_end;
+    if (ip_size < 0)
+    {
```

Comment:
Just nitpicking, but if you touch the code, we usually put the opening line (except for function defs) on the previousl line.

Reply:
Ah, damn. I knew this and yet .. I will fix it.